containers/ecr-viewer/src/app/api/save-fhir-data/save-fhir-data-service.ts

```diff
@@ -106,6 +107,40 @@ export const saveToAzure = async (
   }
 };
 
+/**
+ * Saves a FHIR bundle to Google Cloud Storage.
+ * @param fhirBundle - The FHIR bundle to be saved.
+ * @param ecrId - The unique ID for the eCR associated with the FHIR bundle.
+ * @returns An object containing the status and message.
+ */
+const saveToGCP = async (
+  fhirBundle: Bundle,
+  ecrId: string,
+): Promise<SaveResponse> => {
+  const containerClient = gcpClient();
+  const blobName = `${ecrId}.json`;
+  const body = JSON.stringify(fhirBundle);
+
+  try {
+    await containerClient?.file(blobName).save(body);
```

Comment:
do we need to check if we have a client here? if the container client is nullish I think this await will short circuit to fine and then we'll say we saved it even though we didn't

Reply:
Yes we do!